packages/core/src/codewhispererChat/controllers/chat/messenger/messenger.ts

```diff
@@ -13,10 +13,12 @@ import {
     QuickActionMessage,
 } from '../../../view/connector/connector'
 import { EditorContextCommandType } from '../../../commands/registerCommands'
+import { ChatResponseStream as qdevChatResponseStream } from '@amzn/amazon-q-developer-streaming-client'
 import {
+    ChatResponseStream as cwChatResponseStream,
     CodeWhispererStreamingServiceException,
-    GenerateAssistantResponseCommandOutput,
     SupplementaryWebLink,
+    __MetadataBearer,
```

Comment:
Why important this? Seems like its supposed to be private because of the `__`. Is there something else that can be used or should it be updated to not imply secrecy?

Reply:
You're right, I wanted to look for something that is agnostic to SendMessage and GenerateAssistantResponse response types and this was the minimal common type.

I cannot update the type itself as it's imported from the generated client. I can create a custom type for the properties I need only instead. They're pretty basic